assembly/src/parsers/ast/tests.rs

```diff
@@ -139,6 +139,46 @@ fn test_ast_parsing_use() {
     assert_program_output(source, procedures, nodes);
 }
 
+#[test]
+fn test_ast_parsing_module_multiple_if() {
```

Comment:
For this and the following tests, would be nice to check not only that we can parse the code, but also that the returned AST is correct.